src/main/kotlin/no/nav/medlemskap/regler/v1/Hovedregler.kt

```diff
@@ -156,7 +167,9 @@ class Hovedregler(private val datagrunnlag: Datagrunnlag) {
             return when (resultat.svar) {
                 JA -> jaKonklusjon(ytelse).utfør().copy(harDekning = resultat.harDekning, dekning = resultat.dekning)
                 NEI -> neiKonklusjon(ytelse).utfør().copy(harDekning = resultat.harDekning, dekning = resultat.dekning)
-                UAVKLART -> uavklartKonklusjon(ytelse).utfør().copy(harDekning = resultat.harDekning, dekning = resultat.dekning)
+                UAVKLART ->
+                    uavklartKonklusjon(ytelse).utfør()
+                        .copy(harDekning = resultat.harDekning, dekning = resultat.dekning)
             }
         }
 
```

Comment:
Ser ingen funksjonelle endringer i fila, foreslår at denne fjernes fra commit(s).

/me stemmer for PRs som bare har diff på endringen PRen handler om =P